docs/src/snap/howto/dualstack.md

```diff
@@ -0,0 +1,133 @@
+# How to enable Dual-Stack networking
+
+Dual-stack networking allows Kubernetes to support both IPv4 and IPv6 addresses
+simultaneously. This means that your pods, services can be assigned
+both IPv4 and IPv6 addresses, allowing them to communicate over either protocol.
+This document will guide you through enabling dual-stack, including necessary
+configurations, known limitations, and common issues.
+
+
+### Prerequisites
+
+Before enabling dual-stack, ensure that your environment supports IPv6, and
+that your network configuration (including any underlying infrastructure) is
+compatible with dual-stack operation.
+
+### Enabling Dual-Stack
+
+Dual-stack can be enabled by specifying both IPv4 and IPv6 CIDRs during the
+cluster bootstrap process. The key configuration parameters are:
+
+- **Pod CIDR**: Defines the IP range for pods.
+- **Service CIDR**: Defines the IP range for services.
+
+1. **Bootstrap Kubernetes with Dual-Stack CIDRs**
+
+   Start by creating a new container (e.g., using LXC) and ensure that IPv6
+   is enabled.
+
+   ```
+   lxc launch ubuntu:22.04 k8s-dualstack -p default -p k8s-integration
+   ```
+
+   Shell into the container:
+
+   ```
+   lxc shell k8s-dualstack bash
+   ```
+
+   Now, bootstrap the cluster in interactive mode and set both IPv4 and
+   IPv6 CIDRs:
+
+   ```
+   sudo k8s bootstrap --timeout 10m --interactive
+   ```
+
+   When prompted, set the Pod CIDR and Service CIDR:
+
+   ```
+   Please set the Pod CIDR: [10.1.0.0/16]: 10.1.0.0/16,fd01::/108
+   Please set the Service CIDR: [10.152.183.0/24]: 10.152.183.0/24,fd98::/108
+   ```
```

Comment:
Would be useful to show how to do this with a bootstrap-config.yaml

Reply:
Additionally, I know this is a dualstack document, but what should the user input if they want *only* IPv6?

Reply:
IMHO a How-to should have one straight-forward way to get something done and not list all possible solutions (this can be looked up in the reference) but I'm fine with it if this is the teams opinion.

@eaudetcobello IPv6-only is not yet fully supported, e.g. the Cilium LB does not support that.